modules/ROOT/partials/configuration/images_upload_base_path.adoc

```diff
@@ -11,6 +11,9 @@ Type: `+String+`
 ----
 tinymce.init({
   selector: 'textarea',  // change this value according to your HTML
+  plugins: 'image',
```

Comment:
This is a core option and not tied to the image plugin so we shouldn't be listing this here.

- [ ] Revert this as core options shouldn't have plugins listed. If it's needed for certain pages then it this line should be conditional.

Reply:
Fixed